Jenkinsfiles/prod-integration-tests.yml

```diff
@@ -0,0 +1,9 @@
+pipeline:
+  enabled: true
+  script: "integration-tests"
+job:
+  dockerfile: "DockerfileIntegrationTests"
+  selenium: "2.48.2"
+  selenium_nodes: 5
+  base_url: 'https://developer.mozilla.org'
+  tests: "not login"
```

Comment:
Sorry @glogiotatidis, I suggested the wrong value here. You'll have to use::
   `tests: '"not login"'`
or::
    `tests: "\"not login\""`
instead, since it's an expression.